src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -262,6 +269,18 @@ SscsCaseData convertAppealToSscsCaseData(SyaCaseWrapper appeal) {
         return sscsCaseData;
     }
 
+    private String resolvePostCode(SyaCaseWrapper appeal) {
+        if (appeal.getIsAppointee()) {
+            return Optional.ofNullable(appeal.getAppointee())
+                .map(SyaAppointee::getContactDetails)
+                .map(SyaContactDetails::getPostCode)
+                .filter(appointeePostCode -> !"".equals(appointeePostCode))
```

Comment:
minor: could we use apache common StringUtils isNotEmpty / isNotBlank for this?

Reply:
Oooh of course, how did I not?